### PR TITLE
2740: Add animation to beta nav dropdown

### LINKED ISSUE #2740: Beta: Explore open/close interactions of menus

### PR DESCRIPTION
**Pull request by Caleb-Ellis**
## Done

- Added a slight fade in and vertical movement to beta nav dropdown
- Moved navigation javascript to navigation.js file
- Fixed formatting on Download tab

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Check that the navigation dropdowns animate
- Go to a page with breadcrumbs (e.g. /cloud) and test the animation there too
- Check that the Download tab doesn't have a second resource column

## Issue / Card

Fixes #2740

### CODE REVIEW

File: static/js/navigation.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+var navDropdowns = document.querySelectorAll('.p-navigation__dropdown-link');
+var dropdownWindow = document.getElementsByClassName('dropdown-window')[0];
```
Comment (steverydz) [inline static/js/navigation.js]: Any reason why you haven't used `querySelector` here?

File: static/js/navigation.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    navDropdowns.forEach(function(dropdown) {
+      var dropdownContent = document.getElementById(dropdown.id + "-content");
+
+      if (dropdown === clickedDropdown) {
+        if (dropdown.classList.contains('is-selected')) {
+          dropdown.classList.remove('is-selected');
+          dropdownWindow.classList.add('u-hide--fade');
+          dropdownContent.classList.add('u-hide--fade');
+          dropdownContent.style = "visibility: hidden;"
```
Comment (steverydz) [inline static/js/navigation.js]: Changing styles directly with JavaScript can be quite expensive. Can this be handled with a utility class?

File: static/js/navigation.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        } else {
+          dropdownContent.style = "display: block; visibility: visible;"
+          dropdown.classList.add('is-selected');
+          dropdownContent.classList.remove('u-hide--fade');
+        }
+      } else {
+        dropdown.classList.remove('is-selected');
+        dropdownContent.classList.add('u-hide--fade');
+        dropdownContent.style = "display: none;"
```
Comment (steverydz) [inline static/js/navigation.js]: Same as previous comment

File: static/js/navigation.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      }
+    });
+  });
+});
+
+var globalNav = document.getElementsByClassName('global-nav')[0];
+var globalNavDropdown = document.getElementsByClassName('global-nav__dropdown-link')[0];
+var globalNavContent = document.getElementsByClassName('global-nav__dropdown-content')[0];
```
Comment (steverydz) [inline static/js/navigation.js]: Same as first comment

**Comment by frankiegranato** [comment]
@kwm14 Can you please do this Design Review as @anasereijo doesn't have enough background on this project to do these reviews yet. Thank you.

**Comment by kwm14** [comment]
Looking good @Caleb-Ellis Design 👍

**Pull request merged**